backend/src/main/java/com/bakdata/conquery/io/jackson/serializer/NsIdRef.java

```diff
@@ -21,4 +21,9 @@
 @JsonDeserialize(using = NsIdReferenceDeserializer.class)
 @Target({ElementType.FIELD, ElementType.TYPE_PARAMETER, ElementType.TYPE_USE})
 public @interface NsIdRef {
+	/**
+	 * Sets the scope on which nodes a {@link NamespacedId} should be resolved.
+	 * If a reference is encountered on a node outside the scope (e.g. {@code @NsIdRef(serdesTarget=MANAGER)} on a {@link com.bakdata.conquery.commands.ShardNode}) the field will resolve to {@code null}.
+	 */
+	SerdesTarget serdesTarget () default SerdesTarget.MANAGER_AND_SHARD;
```

Comment:
Das muss nicht jetzt angegangen werden, aber hast du dir hierzu angeschaut, ob/wie Views das abbilden können? Wäre schon schön, wenn wir auf lange Sicht von weniger als mehr NIH machen könnten

Reply:
Was ist NIH?

Reply:
Not invented here